beacon/validator/block_builder.go

```diff
@@ -336,17 +332,7 @@ func (s *Service) buildBlockBody(
 	// Fill in unused field with non-nil value
 	body.SetSyncAggregate(&ctypes.SyncAggregate{})
 
-	// Get the epoch to find the active fork version.
-	epoch := s.chainSpec.SlotToEpoch(blk.GetSlot())
-	activeForkVersion := s.chainSpec.ActiveForkVersionForEpoch(
-		epoch,
-	)
-	if activeForkVersion >= version.DenebPlus {
```

Comment:
note: we never activated DenebPlus so this change is perfectly backward compatible